lib/computer.rb

```diff
@@ -0,0 +1,33 @@
+module TicTacToe
+  class Computer < Player
+    attr_reader :marker
```

Comment:
I believe the `marker` attr_reader is defined in the Player class, so I don't think you should need this line.